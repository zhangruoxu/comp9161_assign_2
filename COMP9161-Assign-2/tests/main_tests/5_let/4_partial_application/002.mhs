main
     =
    let f =
        letfun f x = 
          letfun g y = y + y;

    in f 2 3;
