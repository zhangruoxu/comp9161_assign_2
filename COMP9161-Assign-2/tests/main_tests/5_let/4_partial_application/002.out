main :: Int = let f :: forall a. (a -> Int -> Int) = (letfun f :: (a -> Int -> Int) x = (letfun g :: (Int -> Int) y = (+) y y));
              in f 2 3;
